pkg/payload/incoming/jwt.go

```diff
@@ -0,0 +1,25 @@
+package incoming
+
+import (
+	"github.com/dgrijalva/jwt-go"
+	"github.com/pkg/errors"
+)
+
+type (
+	Token struct {
```

Comment:
Can we be more specific here about what kind of token are we talking about? So `Auth` would probably be a better choice for struct name. And for the file name as well.